src/main/kotlin/io/github/inductiveautomation/kindling/MainPanel.kt

```diff
@@ -139,7 +139,7 @@ class MainPanel : JPanel(MigLayout("ins 6, fill, hidemode 3")) {
                 putClientProperty("FlatLaf.styleClass", "h1")
             },
         )
-        for (tools in Tool.sortedByTitle.chunked(3)) {
+        for (tools in Tool.sortedByTitle.filterNot { it.isAdvanced }.chunked(3)) {
```

Comment:
Just to be clear, we don't want the SerialViewer tool to be its own tool right? It's only being used in the FileViewer tool in the zipviewer?

Reply:
No, this is a standalone top level tool (so I can just point at any arbitrary .bin file on disk easily) 
Purely a convenience thing for developers.

Though, a mode to decode binary files as Java serialized data inside file view is probably also a good idea.

Reply:
Ah I see. I forgot about the File menu. So for now, advanced tools are only accessible from there. It would be cool to add a toggle in the file view to switch between the two views, and shouldn't be too difficult, but this PR is simple and works for now.

Reply:
It'd also be nice to add a toggle to show advanced tool tiles in the advanced preference section.